proto/cacheclient.proto

```diff
@@ -58,53 +58,66 @@ message _SetResponse {
 
 message _DictionaryGetRequest {
   bytes dictionary_name = 1;
-  repeated bytes dictionary_keys = 2;
+  repeated bytes fields = 2;
 }
 
 message _DictionaryGetResponse {
   message _DictionaryGetResponsePart {
     ECacheResult result = 1;
     bytes cache_body = 2;
   }
-  repeated _DictionaryGetResponsePart dictionary_body = 1;
-  string message = 2;
+
+  message _Found {
+    repeated _DictionaryGetResponsePart items = 1;
+  }
+
+  message _Missing {}
+
+  oneof dictionary {
+    _Found found = 1;
+    _Missing missing = 2;
+  }
```

Comment:
I applied the `_Found` (`_Missing`) pattern to the dictionary overall but not the unary items within.